.github/workflows/lsst-tests-ngmix-upstream.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           conda config --set always_yes yes
           mamba install --quiet --yes --file requirements.txt
           mamba install --quiet --yes --file dev-requirements.txt
-          mamba install -q -y stackvana=0 lsstdesc-weaklensingdeblending
+          mamba install -q -y stackvana==0.2024.24 lsstdesc-weaklensingdeblending
```

Comment:
Do we need this pin? Maybe we can use `>=` instead of `==`? Otherwise we are going to miss breakages as the DM code changes.

Reply:
At some point the `=0` stopped working

Reply:
That either indicates the stack APIs have changed or a conda issue. The pin here is hiding that.

Reply:
Yeah, I think what happened is it failed to install with `=0`